### PR TITLE
Closes #143 Refactor code to remove syntax errors and typos

### LINKED ISSUE #143: [FEATURE] Refactor code to remove typos
**Is your feature request related to a problem? Please describe.**
There are a few typos in naming variables and syntax mistakes which are ignored implicitly. One example will be `embededMessage` which must be spelled as `embeddedMessage`. I guess this was copy/pasted, so its spelled in this manner in many modules.

**Describe the solution you'd like**
Would like those and syntax mistakes to be fixed.

**Describe alternatives you've considered**
I guess I can't consider alternatives for this. But yea, I have a PR.

### PR DESCRIPTION
**Pull request by RazCrimson**
Fixes #143
Also has a minor variable correction in `src/commands/poll.js`

### CODE REVIEW

File: src/commands/add-eggs.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const Util = require('../utils/utils.js')
 
 const AddEggs = {
-    check: (item, message) => {
+    check: async (item, message) => {
```
Comment (rgroves) [inline src/commands/add-eggs.js]: This is not meant to be an async function and will cause an issue with the easter egg check in the add function.

File: src/commands/add-eggs.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const AddEggs = {
                     'You found the Konami Code Easter Egg.\nNothing was added.'
                 )
 
-                message.channel.send(embededMessage)
+                await message.channel.send(embeddedMessage)
```
Comment (rgroves) [inline src/commands/add-eggs.js]: Can you explain why you added await here? It should not be required.

File: src/commands/add.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ module.exports = {
         let item = args.join(' ')
         let { channel } = message
 
-        if (easterEggCheck(item, message)) {
+        if (await easterEggCheck(item, message)) {
```
Comment (rgroves) [inline src/commands/add.js]: easterEggCheck should not be made an async function therefore the await keyword should not be required here.

File: src/commands/add-eggs.js (hunk 2 above)
Comment (RazCrimson) [reply src/commands/add-eggs.js]: Aren't promises meant to be awaited?

File: src/commands/add-eggs.js (hunk 1 above)
Comment (RazCrimson) [reply src/commands/add-eggs.js]: I did add a await there. So that should be fixed.

File: src/commands/add-eggs.js (hunk 2 above)
Comment (RazCrimson) [reply src/commands/add-eggs.js]: It works without that. But having a await is better, right?

File: src/commands/add.js (hunk 3 above)
Comment (RazCrimson) [reply src/commands/add.js]: Well I guess, we can remove the await for this and make check a normal function as using a await here feels convoluted.

**Comment by RazCrimson** [comment]
Let me just make a new PR for the typos.

**Pull request closed without merging**